src/import/ImportUtil.ts

```diff
@@ -175,6 +176,13 @@ export default class ImportUtil {
       if (ImportUtil.knownBadProposalCids.includes(deal.ProposalCid['/'])) {
         continue;
       }
+      //Skip over already expired deals
+      let currentHeight = HeightFromCurrentTime();
+      if (deal.Proposal.StartEpoch! < currentHeight) {
```

Comment:
I'd add some buffer time to account for sealing time, and expose that time to the CLI arguments. Do you know how to do so? If not, I'm more than happy to do this for you.

Reply:
Could you please?
You should be able to push into my fork :) 

Still very rusty and re-learning my JS